.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,33 @@
+---
+name: "\U0001F41E Bug report"
+about: Let us know if something isn't working as you expect it to.
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**WordCamp:**
```

Comment:
This is a really good idea. People often report stuff w/out telling us which camp they're working on. 

`MAYBE` - I wonder if it'd be better to ask for a specific URL instead, since they might not make it clear which specific screen they're referring to.

Reply:
I agree, this is awesome, and also a specific URL would be more helpful.